src/sanitize-argument.ts

```diff
@@ -107,6 +120,19 @@ export function sanitize_redis(argv:any)
   }
 }
 
+export async function sanitize_customMainPage(customMainPage: string, baseUrl: string, wikiPath: string) {
+  await axios.get(`${baseUrl}/${wikiPath || 'wiki'}/${customMainPage.replace(/ /g,'_')}`)
+  .then((res)=>{
+    if(res.status !== 200) {
+      throw new Error('Failed to verify status code 200 on custom main page')
+    }
+  })
+  .catch((err) => {
+    throw err;
+  });
+  return customMainPage.replace(/ /g,'_');
+}
+
```

Comment:
This function should be remove as most of the line is the exception mgmt, which is duplicate with lin 61.

We should not have this `wiki` here, this is a default value which should be handled only in the Mediawiki module.

`replace(/ /g,'_')` should not be done twice. Only once.